gammapy/image/utils.py

```diff
@@ -799,7 +833,24 @@ def make_header(nxpix=100, nypix=100, binsz=0.1, xref=0, yref=0,
 
     Parameters
     ----------
-    TODO
+    nxpix : int
+	Number of pixels in x axis. Default is 100.
```

Comment:
No ... I'm not aware of a Sphinx extension that does this automatically.
It would be possible to write one though, but probably difficult to get right in all cases.